integrated_channels/sap_success_factors/models.py

```diff
@@ -200,7 +201,15 @@ def unlink_inactive_learners(self):
         Unlink inactive SAP learners form their related enterprises
         """
         sap_learner_manager = self.get_learner_manger()
-        sap_learner_manager.unlink_learners()
+        try:
+            sap_learner_manager.unlink_learners()
+        except ClientError as exc:
+            LOGGER.error(
+                'Failed to unlink learners for integrated channel [%s] [%s]',
+                self.enterprise_configuration.enterprise_customer.name,
+                self.enterprise_configuration.channel_code,
+            )
+            LOGGER.error(exc)
```

Comment:
`exc` should be appended to previous `LOGGER.error` message.

Reply:
fixed